src/server/api/terrain.js

```diff
@@ -27,18 +27,16 @@ import { replaceNamedPathSegments } from "../../common/functions";
 export const handler = ({ method, pathname, headers }) => {
     return async (req, res) => {
         const accessToken = req?.kauth?.grant?.access_token;
-        call(
-            {
-                method,
-                pathname,
-                headers,
-                query: req.query,
-                params: req.params,
-                userID: accessToken?.content?.preferred_username,
-                accessToken: accessToken?.token,
-            },
-            req
```

Comment:
Previously we were sending the entire request object, which below gets renamed to `inStream`, and was being set as the request body